<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMeshRenderer3dAveragingWidgetBase</class>
 <widget class="QWidget" name="QgsMeshRenderer3dAveragingWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="mAveragingMethodStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mAveragingMethodSingleLevelTopPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="mAveragingMethodLabel_2">
         <property name="text">
          <string>Vertical layer index</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsSpinBox" name="mSingleVerticalLayerIndexTopSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="mSingleTopGroup">
         <property name="title">
          <string>Example: Value of top vertical layer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="mSingleTopPngLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>png</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mSingleTopDescLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Display the value from a vertical layer from the top (surface). The level is truncated to the maximum number of vertical layers for particular face.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mAveragingMethodSingleLevelBottomPage">
      <layout class="QGridLayout" name="gridLayout_17">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="mAveragingMethodLabel_11">
         <property name="text">
          <string>Vertical layer index</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsSpinBox" name="mSingleVerticalLayerIndexBottomSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Example: Value of bottom vertical layer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <widget class="QLabel" name="mSingleBottomPngLabel">
            <property name="text">
             <string>..png...</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mSingleBottomDescLabel">
            <property name="text">
             <string>Display the value from a vertical layer from the bottom (bed). The level is truncated to the maximum number of vertical layers for particular face.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mAveragingMethodMultiTopPage">
      <layout class="QGridLayout" name="gridLayout_20">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="mAveragingMethodLabel_3">
         <property name="text">
          <string>Start vertical layer index</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsSpinBox" name="mMultiTopVerticalLayerStartIndexSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="mAveragingMethodLabel_5">
         <property name="text">
          <string>End vertical layer index</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QgsSpinBox" name="mMultiTopVerticalLayerEndIndexSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Example: first 2 vertical levels </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="mMultiTopPngLabel">
            <property name="text">
             <string>..png...</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mMultiTopDescLabel">
            <property name="text">
             <string>Averages values from multiple vertical layers from the top (surface). The level is truncated to the maximum number of vertical layers for particular face.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mAveragingMethodMultiBottomPage">
      <layout class="QGridLayout" name="gridLayout_21">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="mAveragingMethodLabel_4">
         <property name="text">
          <string>Start vertical layer index</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsSpinBox" name="mMultiBottomVerticalLayerStartIndexSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="mAveragingMethodLabel_6">
         <property name="text">
          <string>End vertical layer index</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QgsSpinBox" name="mMultiBottomVerticalLayerEndIndexSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Example: last 2 vertical levels</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="mMultiBottomPngLabel">
            <property name="text">
             <string>..png...</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mMultiBottomDescLabel">
            <property name="text">
             <string>Averages values from multiple vertical layers from the bottom (bed). The level is truncated to the maximum number of vertical layers for particular face.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mAveragingMethodSigmaPage">
      <layout class="QGridLayout" name="gridLayout_12">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="mAveragingMethodLabel_7">
         <property name="text">
          <string>Start fraction </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsDoubleSpinBox" name="mSigmaStartFractionSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>End fraction</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QgsDoubleSpinBox" name="mSigmaEndFractionSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Example: Sigma range 0.25 - 0.75 </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="mSigmaPngLabel">
            <property name="text">
             <string>..png...</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mSigmaDescLabel">
            <property name="text">
             <string>Sigma averages over the values between 0 (bed level) and 1 (surface).</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mAveragingMethodDepthPage">
      <layout class="QGridLayout" name="gridLayout_13">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="mAveragingMethodLabel_8">
         <property name="text">
          <string>Start depth</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsDoubleSpinBox" name="mDepthStartSpinBox">
         <property name="maximum">
          <double>99999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lab2">
         <property name="text">
          <string>End depth</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QgsDoubleSpinBox" name="mDepthEndSpinBox">
         <property name="maximum">
          <double>99999.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Example: Depth 0.5 to 1.7</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="mDepthPngLabel">
            <property name="text">
             <string>..png...</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mDepthDescLabel">
            <property name="text">
             <string>Depth averaging method averages the values based on range defined relative to surface. The range is defined in the same length units as defined by model (e.g. meters)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mAveragingMethodHeightPage">
      <layout class="QGridLayout" name="gridLayout_24">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="mAveragingMethodLabel_9">
         <property name="text">
          <string>Start height</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsDoubleSpinBox" name="mHeightStartSpinBox">
         <property name="maximum">
          <double>99999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labl">
         <property name="text">
          <string>End height</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QgsDoubleSpinBox" name="mHeightEndSpinBox">
         <property name="maximum">
          <double>99999.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Example: Height 0-1.5m</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QLabel" name="mHeightPngLabel">
            <property name="text">
             <string>..png...</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mHeightDescLabel">
            <property name="text">
             <string>Height averaging method averages the values based on range defined relative to bed elevation. The range is defined in the same length units as defined by model (e.g. meters)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mAveragingMethodElevationPage">
      <layout class="QGridLayout" name="gridLayout_15">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="mAveragingMethodLabel_10">
         <property name="text">
          <string>Start elevation</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsDoubleSpinBox" name="mElevationStartSpinBox">
         <property name="minimum">
          <double>-999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>0.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>End elevation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QgsDoubleSpinBox" name="mElevationEndSpinBox">
         <property name="minimum">
          <double>-999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>0.000000000000000</double>
         </property>
         <property name="value">
          <double>-100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Example: Elevation -0.5 to -2.5</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="mElevationPngLabel">
            <property name="text">
             <string>..png...</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mElevationDescLabel">
            <property name="text">
             <string> Elevation averaging method averages the values based on range defined absolute value to the model's datum. The range is defined in the same length units as defined by model (e.g. meters). The elevation will be truncated at the surface and bed levels.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mAveragingMethodLabel">
     <property name="text">
      <string>Method</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="mAveragingMethodComboBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Single Vertical Level (from top)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Single Vertical Level (from bottom)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multi Vertical Level (from top)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multi Vertical Level (from bottom)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sigma</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Depth (relative to surface)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Height (relative to bed level)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Elevation (absolute to model's datum)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
